app/src/main/res/values/strings.xml

```diff
@@ -99,4 +99,6 @@
     <string name="notes_empty_error">Notes cannot be empty</string>
     <string name="available_to_be_mentor_mentee">Available to be a:</string>
     <string name="settings">Settings</string>
+    <string name="verify_email">Please verify your email before login.</string>
```

Comment:
```suggestion
    <string name="verify_email">Please verify your email first.</string>
```

Reply:
Do you need it to be the same message because of your string comparison? If you create two separate variables for the observable LiveData variables `successful` you can have one behavior assigned to each listener. And you won't need to compare the string to know if you are in a resend email situation

Reply:
@isabelcosta I tried creating different LiveData variables but then also I have to compare the strings.
See, when user tried to press Login Button without verifying the email then the value of "successful" variable is false and the message variable will contain that message which is coming from backend. And in the else condition only (when the value of successful is false) I am comparing that message with "loginViewModel.message" if they both are equal then I am showing Snackbar with "Resend email" option and when user clicks "Resend email" option then I am setting value of "successful" equals to false which means that it will go in else condition in onCreate() menthod in LoginActivity and will show another Snackbar saying "New confirmation email has been sent"

Reply:
I did not want the message to be changed on Backend.
What I am saying is that I tried creating different LiveData variables but, to know that we are in a resend email situation I need to compare the strings.

Reply:
Will leave this probably to another separate issue, we can put this improvement in "icebox" stage, and if someone is interested in this can pick up